tests/flow/test_concurrent_query.py

```diff
@@ -172,22 +145,25 @@ def test_05_concurrent_read_delete(self):
         # Delete graph via Redis DEL key.
         ##############################################################################################
         self.populate_graph()
+        pool = Pool(nodes=CLIENT_COUNT)
+        indexes = range(CLIENT_COUNT)
+
         q = """UNWIND (range(0, 10000)) AS x WITH x AS x WHERE (x / 900) = 1 RETURN x"""
-        threads = []
-        for i in range(CLIENT_COUNT):
-            graph = graphs[i]
-            t = threading.Thread(target=thread_run_query, args=(graph, q, i))
-            t.setDaemon(True)
-            threads.append(t)
-            t.start()
+        queries = [q] * CLIENT_COUNT
+        # invoke queries
+        m = pool.amap(thread_run_query, graphs, queries, indexes)
 
         redis_con.delete(GRAPH_ID)
```

Comment:
This test need to be revisit, we're querying different graphs but are trying to delete only a single graph
I think the idea is to try and delete a graph which is currently being queried multiple times